### DISCUSSION TITLE
What is the '2023' on the left side of the blog?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#5847] [Q&A] [answered]
<img width="166" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/62934294/4b1814ae-e20c-487b-95f1-eba1d924d115">

What is the 2023 section on the left side of the blog? How to hide it? Thanks!

**Comment by squidfunk** [comment]
Not sure, please provide a self-contained, [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/). We're not seeing it on our documentation.

<img width="170" alt="Bildschirm­foto 2023-08-16 um 07 57 04" src="https://github.com/squidfunk/mkdocs-material/assets/932156/07e42764-0b06-431f-8e1f-13192b9d4153">

**Reply by mendax1234** [reply]
Hi, thanks for your reply. l think it is better to look at my repo, it is [here](https://github.com/mendax1234/mendax1234.github.io). Thank you so much!

**Reply by squidfunk** [reply]
[You should not link your posts in `mkdocs.yml`](https://github.com/mendax1234/mendax1234.github.io/blob/f7e4f283725df349689c39c26236f399e033d203/mkdocs.yml#L89-L96)

**Reply by mendax1234** [reply]
Thanks! Where should l link my posts? [l think your posts are also linked in the `mkdocs.yml`](https://github.com/squidfunk/mkdocs-material/blob/master/mkdocs.yml#L232C1-L240C51) 😂

**Reply by squidfunk** [reply, marked as answer]
You should not put them in `nav`. This was necessary before the blog plugin was in the community edition, so MkDocs did not complain. You should only provide the `blog/index.md`, [as mentioned in our documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#__codelineno-2-1:3).

**Reply by mendax1234** [reply]
l see. Thanks a lot!
